frontend/javascripts/oxalis/view/right-menu/meshes_view.js

```diff
@@ -158,6 +227,15 @@ class MeshesView extends React.Component<Props, { currentlyEditedMesh: ?MeshMeta
             </Tooltip>
           )}
         </ButtonGroup>
+
+        <List
+          dataSource={Object.values(this.props.isosurfaces)}
+          size="small"
+          split={false}
+          renderItem={renderListItem}
+          locale={{ emptyText: " " }}
```

Comment:
Here you can put some small explanation of the list (i.e., what is displayed and how can the user add something to this list).